shared_model/interfaces/base/signable.hpp

```diff
@@ -88,6 +88,26 @@ namespace shared_model {
             and this->createdTime() == rhs.createdTime();
       }
 
+      /**
+       * Provides hash from payload
+       * @return hash of payload
+       */
+#ifndef DISABLE_BACKWARD
```

Comment:
Minimize code wrapped in that macro plz (using with the Hashable type)

Reply:
#1022

Reply:
merged and fixed